tests/fread/test-fread-large.py

```diff
@@ -106,6 +106,25 @@ def f(request):
 # Run the tests
 #-------------------------------------------------------------------------------
 
+
+# Make sure we can read/write csv and jay files larger than 4Gb on Windows
+# @pytest.mark.usefixtures("winonly", "is_release")
+@pytest.mark.usefixtures("winonly")
+def test_fread_4gb_plus(tempfile):
+    size = 5 * 10**6
+    DT0 = dt.Frame([True] * size)
+    DT0.to_jay(tempfile)
+    assert(os.path.getsize(tempfile) > size)
```

Comment:
`assert` is not a function, so there shouldn't be any parentheses

Reply:
yeah, a leftover from the edited `assert_equals()`...